configuration/pih/htmlforms/labResults.xml

```diff
@@ -104,8 +104,14 @@
             }
 
             jq(function() {
-                getField('specimen-collection-date.value').datepicker('option', 'maxDate', new Date());
+                getField('encounter-date.value').change(function(evt) {
+                    setValue('specimen-collection-date.value', evt.target.value);
+                    getField('test-date.value').datepicker('option', 'minDate', evt.target.value);
+                });
+                setValue('specimen-collection-date.value', new Date());
+                getField('encounter-date.value').datepicker('option', 'maxDate', new Date());
                 getField('test-date.value').datepicker('option', 'maxDate', new Date());
+                getField('test-date.value').datepicker('option', 'minDate', new Date());
```

Comment:
Does this work properly during edit? ie, if you are are opening an encounter that has a both a encounter date and a test date in the past?

Reply:
It didn't, but it does now